bin/strings-export.js

```
@@ -0,0 +1,44 @@
+#! /usr/bin/env node
+"use strict";
+
+/* eslint-disable no-console */
+
+const TEMPLATES_PATH = "l10n/templates";
+
+/* globals cp */
+require("shelljs/global");
+
+// Get a reference to the local repository
+const git = require("simple-git");
+const localRepo = git();
+
+// Ues the first script argument as the target repository of strings
+const [, , stringsPath] = process.argv;
+const stringsRepo = git(stringsPath);
```

should we maybe throw early if `stringsPath` is missing?